float32 resolution
string destination
bool saveMap
bool visualMap
---
bool success
